real_time/surface_src/java_src/Alice/src/test/java/TestWayPointFollowerPlanner.java

```diff
@@ -28,24 +30,25 @@ public void test() {
 			fail("distance function does not respect idenity");
 		}
 		
-		if(Math.abs(GPSPoseMessage.getDistance(zeroPose, onePose) - 1.0) > .001){
+		if(Math.abs(GPSPoseMessage.getDistance(zeroPose, onePose) - LocalizerUtil.convertLonToMeters(1.0)) > .001){
 			fail("distance function does not handle some input correctly ");
 		}
 		
-		if(Math.abs(GPSPoseMessage.getDistance(aPose, zeroPose) -  1.802) > .001){
+		if(Math.abs(GPSPoseMessage.getDistance(onePose, aPose) - LocalizerUtil.convertLatToMeters(1.5)) > .001){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2362883654/issues/source?bid=2938665&fileBranchId=3216826#l37)